label_studio/io_storages/synology/models.py

```diff
@@ -119,21 +119,23 @@ def iterkeys(self):
             if regex and not regex.match(key):
                 logger.debug(key + ' is skipped by regex filter')
                 continue
-            yield file["path"]
+            yield self.resolve_uri_interal(file["path"])
 
     def get_data(self, key):
         if self.use_blob_urls:
             data_key = settings.DATA_UNDEFINED_NAME
-            return {data_key: f"{self.clean_url}{key}"}
+            return {data_key: f"{key}"} #{self.clean_url}{key}
         
         filestation = self.get_filestation()
         bytes: io.BytesIO = filestation.get_file(key, "serve")
 
         return json.loads(bytes.read())
     
     def resolve_uri(self, uri, task=None):
```

Comment:
Please remove this method.

Reply:
doing so causes the project to fail to load in data. The base class will do undesired behavior with the Synology uri. This passthrough function allows us to prevent that